packages/fxa-settings/src/models/contexts/AppContext.ts

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ApolloClient } from '@apollo/client';
+import AuthClient from 'fxa-auth-client/browser';
+import React from 'react';
+import config, { Config, readConfigMeta, getDefault } from '../../lib/config';
+import { createApolloClient } from '../../lib/gql';
+import { Account } from '../Account';
+import { Session } from '../Session';
+
+// TODO, move some values from AppContext to SettingsContext after
+// using container components, FXA-8107
+export interface AppContextValue {
+  authClient?: AuthClient;
+  apolloClient?: ApolloClient<object>;
+  config?: Config;
+  account?: Account;
+  session?: Session; // used exclusively for test mocking
```

Comment:
Could you elaborate more on this comment?

Reply:
This was an old comment, so not super sure. Might become more obvious as I'm finishing up tests but we can take a look in FXA-8107 otherwise.